src/main/java/scrolls/elder/model/person/Person.java

```diff
@@ -29,12 +29,13 @@ public abstract class Person {
     protected final Set<Tag> tags = new HashSet<>();
     protected final Optional<Name> pairedWithName;
     protected final Optional<Integer> pairedWithId;
+    protected final int timeServed;
```

Comment:
If the timeServed property is only applicable for volunteers, is it necessary to have it implemented in the parent Person class?